src/components/Booklist.js

```diff
@@ -1,31 +1,18 @@
+import { useSelector } from 'react-redux';
+// import { add, remove } from '../redux/books/booksSlice';
 import Book from './Book';
```

Comment:
- [ ] Kindly remove commented (unused) code from your codebase, because it's not considered a good practice to have commented (unused) code in your app.

Reply:
@Omar-Muhamad Thanks, you are right !!!, I forget to delete that. I'll do it right now, thanks for your review :handshake: